http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -101,8 +105,9 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
         case x: CloudContext.Azure => F.pure(x.value)
       }
       hcName = RelayHybridConnectionName(params.runtime.runtimeName.asString)
-      primaryKey <- azureRelay.createRelayHybridConnection(params.relayeNamespace, hcName, cloudContext)
+      primaryKey <- azureRelay.createRelayHybridConnection(params.landingZoneResources.relayNamespace, hcName, cloudContext)
```

Comment:
I thought hybrid connection is also created somewhere now? or maybe I'm misremembering things?

Reply:
We talked about making it a WSM resource but it never actually happened.. so for now Leo is still creating it directly (which I think is fine).

Reply:
yea that's what i was thinking....fwiw, I think it's totally fine to be managed by Leo